app/src/main/java/com/github/HumanLearning2021/HumanLearningApp/firestore/FirestoreDatasetInterface.kt

```diff
@@ -50,6 +50,22 @@ class FirestoreDatasetInterface(
         }
     }
 
+    override suspend fun getRepresentativePicture(category: Category): CategorizedPicture {
+        TODO("Not yet implemented")
+    }
+
+    override suspend fun getAllPictures(category: Category): Set<CategorizedPicture> {
+        TODO("Not yet implemented")
+    }
```

Comment:
This will give us all the pictures of the same category in this dataset right? If so, it's perfect for me.

Reply:
Yes exactly